### ISSUE TITLE
open.spotify.com data is fetched but (still) won't scrobble

### ISSUE THREAD

**Issue by iwconfig** [david-sabata/web-scrobbler#1270]
### Expected Behavior
Well, `testReporter.js` outputs data objects in the console and the same data should be scrobbled. I have installed the latest from the master branch, which includes 1b2b536.
### Current Behavior
`testReporter.js` outputs data objects in the console but no data is actually scrobbled to last.fm. This issue might be similar to #1267.

### Steps to Reproduce (for bugs)
1. Download latest master version from git
2. Activate extension.
3. Uncheck every site _except Spotify (open)_.
4. Re-authenticate with Last.FM.
5. Go to https://open.spotify.com and hit **Ctrl-Shift+R**.
6. Extension recognizes open.spotify.com but nothing happens.

Pastebins of with console output of:
 - Extension: https://pastebin.com/raw/XtF4CQeN
 - open.spotify.com: https://pastebin.com/raw/iV9ZBXqJ

### Context
I'm trying to scrobble my lovely music taste.

### Steps for resolving the issue
- [x] Reauthenticate on Last.fm
- [x] Try to use unpacked extension from latest master
- [x] Check other issues for duplicates

### Environment

* Version used: The unpacked version from master branch gives **1.62.0** but the extension from the web store gives **1.62.2**.
* Browser Name and version: Google Chrome **58.0.3029.31** (Official Build) dev (64-bit)

**Comment by alexesprit** [comment]
Can't reproduce using steps described in your comment. Have you any additional information?

**Comment by iwconfig** [comment]
I'm sorry, I can't think of anything more to add. What would that be?

**Comment by alexesprit** [comment]
@iwconfig Could you install the extension from this zip: https://github.com/alexesprit/web-scrobbler/archive/spotify-test.zip and check if the extension works?

Also, could you post the HTML code of play button? Something like this one:
```html
<button class="control-button spoticon-pause-16 control-button--circled" title="Pause"></button>
```

**Comment by alexesprit** [comment]
Also, in the logs of content scripts this line is truncated:
```
testReporter.js:57	Web Scrobbler: connector_state_changed Object {track: "Do Dekor", artist: "Jan Jelinek", album: null, uniqueID: null, duration: 0…}
```
Could you also expand the line and post its content?

**Comment by iwconfig** [comment]
@alexesprit Confirmed! _spotify-test.zip_ works as it should. Awesome! :)

Its the same HTML code for me.

	<button class="control-button spoticon-play-16 control-button--circled" title="Spela upp"></button>
	<button class="control-button spoticon-pause-16 control-button--circled" title="Pausa"></button>

Here's the whole connector_state_changed Object:
![screenshot 2017-04-04 at 7 37 05 pm](https://cloud.githubusercontent.com/assets/10843441/24670459/5c4e5ec2-196e-11e7-8a14-4eb818453c3f.png)

**Comment by alexesprit** [comment]
@iwconfig thanks for report. This issue is fixed in #1272, but that PR isn't merged yet.

**Comment by iwconfig** [comment]
@alexesprit hey, my pleasure. Oh it was that one, alright. Great, thanks! 👍
